api/src/main/resources/schema/produce/produceevent.json

```diff
@@ -8,7 +8,8 @@
       "minLength": 1
     },
     "data": {
-      "type": "string",
+      "type": "object",
```

Comment:
According to the spec this can be either a string or an Object.
If this shouldn't be a string, can you update the description to reflect the change?

> If object type, a custom object to become the data (payload) of produced event.


https://github.com/serverlessworkflow/specification/blob/0.8.x/specification.md#producedevent-definition

Reply:
perhaps this description needs to be updated as well:
https://github.com/serverlessworkflow/sdk-java/pull/561/files#diff-a1593297ec9764a50a83b736ce196d5272b2b4281ae73613f10d742598638568R21

Reply:
This schema is just an utility to make the generator work. In order the generator to work for both string and object, the generated POJO should return a JsonNode.